sql-server-2016-fci-existing-vnet-and-ad/.ci_skip

```diff
@@ -0,0 +1,2 @@
+Skip CI due to existing resource requirements
```

Comment:
See: https://github.com/Azure/azure-quickstart-templates#template-pre-requisites for how to handle prereqs